package.json

```diff
@@ -6,6 +6,10 @@
   "scripts": {
     "start": "npm run build && node dist/index.js",
     "build": "tsc",
+    "db:create": "node dist/db/tasks/create.js",
```

Comment:
@guieiras Since we're executing the `/dist` files, I think it makes sense to automatically `build` beforehand, don't you think?